### PR TITLE
[flutter_runner] Port over all the changes to the dart_runner cmx files

### PR DESCRIPTION
**Pull request by iskakaushik**

### CODE REVIEW

**Comment by iskakaushik** [comment]
Note to the reviewer: the formatting and lexicographical sorting is out of order for a few of the files. I went for 100% match between topaz and our repos. We can clean these up later.

File: shell/platform/fuchsia/dart_runner/integration/meta/dart_aot_runner_test.cmx [hunk 1]
```diff
@@ -3,14 +3,12 @@
...
     "sandbox": {
-        "features": [
-            "deprecated-ambient-replace-as-executable"
-        ],
         "services": [
             "fuchsia.cobalt.LoggerFactory",
             "fuchsia.fonts.Provider",
             "fuchsia.logger.LogSink",
             "fuchsia.modular.Clipboard",
+            "fuchsia.modular.ContextWriter",
```
Comment (dnfield) [inline shell/platform/fuchsia/dart_runner/integration/meta/dart_aot_runner_test.cmx]: I believe this is deprecated and not needed. but we can remove that later if this is still in topaz.

File: shell/platform/fuchsia/dart_runner/vmservice/meta/vmservice.cmx [hunk 2]
```diff
@@ -20,4 +18,4 @@
             "fuchsia.wlan.service.Wlan"
         ]
     }
-}
+}
```
Comment (dnfield) [inline shell/platform/fuchsia/dart_runner/vmservice/meta/vmservice.cmx]: new line at EOF

Comment (iskakaushik) [reply shell/platform/fuchsia/dart_runner/vmservice/meta/vmservice.cmx]: done.

File: shell/platform/fuchsia/dart_runner/integration/meta/dart_aot_runner_test.cmx (hunk 1 above)
Comment (iskakaushik) [reply shell/platform/fuchsia/dart_runner/integration/meta/dart_aot_runner_test.cmx]: yeah, letting it be for now.

**Comment by dnfield** [comment]
(LGTM)

**Pull request merged**